index.php

```diff
@@ -527,11 +527,19 @@ class="dt">&#148;</span>
 				$rating = explode(" ", $photos[$p]);
 				$hits = $rating[0];
 				$hits++;
+				//secondary version:
+				$rate = 0;
+				$raters = substr(strrchr($rating[1], '/'), 1);
+				if ($raters != "0")
 				@eval("@\$rate =".$rating[1].";");

```

Comment:
indentation maybe? (Now that it's in an `if`)

Reply:
Sorry, I just added it.